website/source/docs/jobops/inspecting.html.md

```diff
@@ -0,0 +1,169 @@
+---
+layout: "docs"
+page_title: "Operating a Job: Inspecting State"
+sidebar_current: "docs-jobops-inspection"
+description: |-
+  Learn how to inspect a Nomad Job.
+---
+
+# Inspecting state
+
+Once a job is submitted, the next step is to ensure it is running. This section
+will assume we have submitted a job with the name "example".
+
+To get a high-level over view of our job we can use the [`nomad status`
+command](/docs/commands/status.html). This command will display the list of
+running allocations, as well as any recent placement failures. An example below
+shows that the job has some allocations placed but did not have enough resources
+to place all of the desired allocations. We run with `-evals` to see that there
+is an outstanding evaluation for the job:
+
+```
+$ nomad status example
+ID          = example
+Name        = example
+Type        = service
+Priority    = 50
+Datacenters = dc1
+Status      = running
+Periodic    = false
+
+Evaluations
+ID        Priority  Triggered By  Status    Placement Failures
+5744eb15  50        job-register  blocked   N/A - In Progress
+8e38e6cf  50        job-register  complete  true
+
+Placement Failure
+Task Group "cache":
+  * Resources exhausted on 1 nodes
+  * Dimension "cpu exhausted" exhausted on 1 nodes
+
+Allocations
+ID        Eval ID   Node ID   Task Group  Desired  Status
+12681940  8e38e6cf  4beef22f  cache       run      running
+395c5882  8e38e6cf  4beef22f  cache       run      running
+4d7c6f84  8e38e6cf  4beef22f  cache       run      running
+843b07b8  8e38e6cf  4beef22f  cache       run      running
+a8bc6d3e  8e38e6cf  4beef22f  cache       run      running
+b0beb907  8e38e6cf  4beef22f  cache       run      running
+da21c1fd  8e38e6cf  4beef22f  cache       run      running
+```
+
+In the above example we see that the job has a "blocked" evaluation that is in
+progress. When Nomad can not place all the desired allocations, it creates a
+blocked evaluation that waits for more resources to become available. We can use
+the [`eval-status` command](/docs/commands/eval-status.html) to examine any
+evaluation in more detail. For the most part this should never be necessary but
+can be useful to see why everything was not placed. For example if we run it on
+the evaluation that had placement failures we see:
```

Comment:
"the evaluation" -> "the _example_ job, which had a placement failure according to the above output, we see:"